README.md

```diff
@@ -14,7 +14,9 @@ We welcome all contributions to improve the materials! Maintainers will do their
 questions, concerns, or experience any difficulties along the way.
 To edit the user guide:
 
-We'd like to ask you to familiarize yourself with our [Contribution Guide](CONTRIBUTING.md).
+We'd like to ask you to familiarize yourself with our [Contribution Guide](CONTRIBUTING.md).  
+
+When contributing, please make changes to the main branch, not the release branch. Once we collect many changes from the main branch, we merge them into the release branch. The current user guide is published from the release branch.
```

Comment:
Maybe that should go to contributing? So we can add more info about the release process as well if needed